pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -4650,6 +4650,34 @@ The matrix transformation will be applied as follows for each coordinate:
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+	"st_rotatex": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"g", types.Geometry},
+				{"angle_radians", types.Float},
+			},
+			ReturnType: tree.FixedReturnType(types.Geometry),
+			Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				g := tree.MustBeDGeometry(args[0])
+				rotRadians := float64(tree.MustBeDFloat(args[1]))
+
+				ret, err := geomfn.RotateX(g.Geometry, rotRadians)
+				if errors.Is(err, geomfn.ErrPointOriginEmpty) {
```

Comment:
can you rearrange this to be:

```
	if err != nil {
       if errors.Is(err, geomfn.ErrPointOriginEmpty) {
          return tree.DNull, nil
			}
			return nil, err
   }
```

normally we're used to seeing `if err != nil` straight up, so it's nicer to read this way

Reply:
Would be acceptable to just omit
```					
if errors.Is(err, geomfn.ErrPointOriginEmpty) {
    return tree.DNull, nil
}
``` 
And keep  
```
if err != nil {
    return nil, err
}
```
I think for this functionality it isn't necessary to check if point origin is empty, now that I take a closer look. Correct me if I'm wrong.

Reply:
Just realized the point origin empty check is not needed actually, sorry about that. Should be okay now https://github.com/cockroachdb/cockroach/pull/61326#discussion_r585891554